scripts/release/rpm/test_rpm_in_docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# This script should be run in a centos8 docker.
+set -exv
+
+yum --nogpgcheck localinstall /mnt/yum/$YUM_NAME -y
+
+yum install git -y
+
+ln -s /usr/bin/python3 /usr/bin/python
+ln -s /usr/bin/pip3 /usr/bin/pip
+time az self-test
+time az --version
+
+cd /azure-cli/
+pip3 install wheel
+./scripts/ci/build.sh
+pip3 install pytest --prefix /usr/lib64/az
+pip3 install pytest-xdist --prefix /usr/lib64/az
+
+find /azure-cli/artifacts/build -name "azure_cli_testsdk*" | xargs pip3 install --prefix /usr/lib64/az --upgrade --ignore-installed
```

Comment:
is it necessary to pin version also?

Reply:
`azdev` uses `pytest>=4.4.0`, which would always install the latest version in CI enviroment and achieves the same effect as no version specified.
Maybe we should first pin it in `azdev` to avoid breaking changes with `pytest`. @haroldrandom 

https://github.com/Azure/azure-cli-dev-tools/blob/5284eaf69b90ab61cc251e55056b040209f12626/setup.py#L69